### ISSUE TITLE
NTR: 5-prime monophosphate hydrolase low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8331]
New term request 5-prime monophosphate hydrolase low in blood (elthat_130418041218): UMPH1; pyrimidine 5-prime nucleotidase

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000240
- label: Reduced circulating pyrimidine 5-prime-nucleotidase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:11369620
- def: The activity of concentration of pyrimidine 5-prime-nucleotidase (EC 3.1.3.5) in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8331
- synonyms:
